### PR TITLE
Add container scanning

### PR DESCRIPTION
**Pull request by farodin91**
Part of #60

* Add security scanning using AquaSecurity Trivy
* Security scan results can be found here: https://github.com/JanusGraph/janusgraph-docker/security/code-scanning
* Use openjdk:8-jre-slim-buster instead of openjdk:8-jdk

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

### CODE REVIEW

**Comment by farodin91** [comment]
I think it would be helpful to make announce after this PR and tagging policy about current changes to our docker images.

**Comment by farodin91** [comment]
Block by #73

File: 0.3/Dockerfile [hunk 1]
```diff
@@ -16,14 +16,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM openjdk:8-jdk as builder
+FROM debian:buster as builder
```
Comment (mbrukman) [inline 0.3/Dockerfile:19]: For even more efficiency, you can maybe use `debian:buster-slim` for the build step, if it has everything you need.

**Review by mbrukman — APPROVED** [review]
LGTM

**Pull request merged**